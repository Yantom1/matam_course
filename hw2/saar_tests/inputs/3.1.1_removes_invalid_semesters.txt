10000000, stude, 20, 2000, 0
20000000, stud, 19, 2001, -1
30000000, stu, 18, 2002, 5
40000000, st, 17, 2003, 100000000
50000000, s, 16, 2004, -100
