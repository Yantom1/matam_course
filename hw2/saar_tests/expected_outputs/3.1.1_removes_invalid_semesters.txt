30000000, stu, 18, 2002, 5
40000000, st, 17, 2003, 100000000
